### PR TITLE
Restructure content management guide part 2

### PR DESCRIPTION
**Pull request by maximiliankolb**
This PR depends on https://github.com/theforeman/foreman-documentation/pull/729

I will rebase to master once the first PR has been merged.

### CODE REVIEW

**Comment by melcorr** [comment]
Only thing I just thought of now, is that {context} tag.  If we could add that everywhere in the titles, that means we could build in multiple locations. Removing that in this PR from the lifecycle environment title, for example, should break that section from appearing in the Installation guide.

**Comment by maximiliankolb** [comment]
I've only removed `_{context}` a couple of times, but I've reverted this by hand.

I've also fixed a couple of unreleated xrefs and rebased to master.

**Comment by maximiliankolb** [comment]
👀

**Comment by melcorr** [comment]
I will try finish these today @maximiliankolb !

**Comment by melcorr** [comment]
OK I am here now, and reviewing. 
Sorry, sorry, sorry

File: guides/common/modules/proc_promoting-a-content-view-across-all-life-cylce-environments-within-an-organization.adoc [hunk 1]
```diff
@@ -0,0 +1,24 @@
+[[Promoting_a_Content_View_Across_All_Life_Cycle_Environments_within_an_Organization]]
```
Comment (melcorr) [inline guides/common/modules/proc_promoting-a-content-view-across-all-life-cylce-environments-within-an-organization.adoc:1]: typo in the file name

**Review by melcorr — CHANGES_REQUESTED** [review]
Is there any way I could convince you to add a _{context} to all these file ids? 
It's a non blocker, but it would be extremely nice to have. 
Just spotted one typo. Everything builds fine, everything looks fine, and I checked all xrefs etc.

File: guides/common/modules/proc_promoting-a-content-view-across-all-life-cylce-environments-within-an-organization.adoc (hunk 1 above)
Comment (maximiliankolb) [reply guides/common/modules/proc_promoting-a-content-view-across-all-life-cylce-environments-within-an-organization.adoc]: Fixed.

**Comment by maximiliankolb** [comment]
Rebased to master; fixed type; added `_{context}` to all (new) anchors; and adjusted the links accordingly. I can squash before merging.

**Comment by maximiliankolb** [comment]
Thanks @melcorr 
Rebased to master; IMHO ready to get merged.

**Pull request merged**